Sources/SwiftSyntaxBuilder/SyntaxNodeWithBody.swift

```diff
@@ -132,6 +173,30 @@ public extension IfExprSyntax {
     self.elseKeyword = elseBody != nil ? .keyword(.else) : nil
   }
 
+  /// Constructs an `if` expression with a following `else if` clause.
+  /// This can be used to create longer chains of `if`, `else if` expressions. For example to construct
+  ///
+  /// ```swift
+  /// if x == 1 {
+  ///     return "one
+  /// } else if x == 2 {
+  ///     return "two
+  /// } else {
+  ///     return "many
+  /// }
+  /// ```
+  ///
+  /// You can use
```

Comment:
I like how this one has an actual example of how to construct it. How would you feel about adding something similar to the others? If not that's fine, some documentation is better than none :). But a lot of the `For example the following ... has ...` could just be code instead.

Reply:
Added more examples